src/commons/utils/constants.ts

```diff
@@ -134,6 +134,8 @@ export const EXT_ADA_PRICE_URL =
   process.env.REACT_APP_EXT_ADA_PRICE_URL || get(window, "env.REACT_APP_EXT_ADA_PRICE_URL");
 export const BOLNISI_NAME_API = process.env.REACT_APP_BOLNISI_NAME_API || get(window, "env.REACT_APP_BOLNISI_NAME_API");
 export const API_GECKO = process.env.REACT_APP_API_URL_COIN_GECKO || get(window, "env.REACT_APP_API_URL_COIN_GECKO");
+export const IS_CONWAY_ERA =
```

Comment:
I would keep FF_GLOBAL_ just to remember this is a feature flag to remove. any thoughts?